### PR TITLE
ENDOC-330 reorganization of intro pages content

### PR DESCRIPTION
**Pull request by jyunmitch**
will fix the links in the next round. it's from moving the development concepts content to a different location.

### CODE REVIEW

File: vuepress/docs/v7.0/docs/getting-started/README.md [hunk 1]
```diff
@@ -41,21 +29,20 @@ https://multipass.run/#install
...
+Manual installation allows you to configure a shared cluster or customize a local developer environment. The following learn-as-you-go approach gives you a working knowledge of Kubernetes as you get Entando up and running in a local environment. After downloading the necessary files, your instance of Kubernetes will be up and running in <60 seconds.
+
+This is a three stage process:
 
 1. [Install Kubernetes](#install-kubernetes)
 2. [Prepare the Kubernetes Environment](#prepare-kubernetes)
 3. [Deploy Entando](#deploy-entando)
 
-
-We've tested a variety of Kubernetes implementations, including Minikube, Minishift, CodeReady Containers, K3s, and Microk8s, to find the optimal combination of low CPU/memory usage, fast startup times, and minimal configuration. After downloading the necessary files, we'll have our own instance of Kubernetes up and running in <60 seconds.
```
Comment (jyunmitch) [inline vuepress/docs/v7.0/docs/getting-started/README.md:57]: @nshaw, Not sure if removing this was a good idea, just thinking that a tutorial should keep peripheral information to a minimum.

File: vuepress/docs/v7.0/docs/getting-started/concepts-overview.md [hunk 2]
```diff
@@ -211,36 +205,36 @@ In this table you can see what are the default routes that are exposed for each
...
 
-Once the link between the Entando App and the Entando Plugin is created, the controller reads the specification of the link and automatically creates the HTTP routes in the Entando App so that the plugin is exposed on the same domain as the App builder, App engine and Component Manager.
+Once the link between the EntandoApp and the plugin is created, the controller reads the specification of the link and automatically creates the HTTP routes in the EntandoApp so that the plugin is exposed in the same domain as the App Builder, App Engine and Component Manager.
 
-This allows the micro frontend developers, who need access to the plugin, to disregard CORS policy and the full path where the plugin is exposed. The plugin can be referenced using relative URLs.
+This allows micro frontend developers, who need access to the plugin, to disregard CORS policy and the full path where the plugin is exposed. The plugin can be referenced using relative URLs.
 
-### How to Check Ingresses in my Cluster
+### How to Check Ingresses in My Cluster
```
[the remark this answers is not in the document]
Comment (Lyd1aCla1r3) [reply vuepress/docs/v7.0/docs/getting-started/concepts-overview.md:217]: lol i would!

File: vuepress/docs/v7.0/docs/getting-started/README.md (hunk 1 above)
Comment (Lyd1aCla1r3) [reply vuepress/docs/v7.0/docs/getting-started/README.md:57]: @jyunmitch i had the same thoughts here; i thought i'd changed it to read "we selected k3s to use the optimal combination..."
i don't think we need to dive into process and trial and error, just discuss what's topical and pertinent

File: vuepress/docs/v7.0/docs/README.md [hunk 3]
```diff
@@ -5,28 +5,56 @@ sidebarDepth: 0
...
-::: tip How Google Runs Containers in Production
-Each week, Google deploys over 2 billion containers in production to run all of its services like Gmail, YouTube, and Search. Kubernetes is Google's open source solution to automate the management of containers at scale.
-:::
-
-Over 50% of Fortune 500 companies use Kubernetes with product offerings from Amazon Elastic Kubernetes Service (EKS), Google Cloud Platform (GKE), IBM (Red Hat) OpenShift, Microsoft Azure (AKS), and VMWare Pivotal (PKS).
+It is a distributed and composable environment in which components can be built, scaled and iterated independently, helping teams build agile applications that can adapt to change.
+
+![Entando Platform Development Process](./getting-started/img/concepts.png) 
+
+On the Entando Platform, development of an application is organized into four stages: Create, Curate, Compose, and Consume. In the simplest terms, you create functional building blocks, curate them into a catalog, compose the blocks onto an app or website, which is then consumed by a user.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/v7.0/docs/README.md:12]: "...catalog, compose the blocks onto an app or website, which..." -> "...catalog, and compose the blocks into an app or website, which..."

File: vuepress/docs/v7.0/docs/README.md [hunk 4]
```diff
@@ -5,28 +5,56 @@ sidebarDepth: 0
...
+
+| Documentation | Tutorials        |
+| :------------ | :--------- |
+| [Entando Component Generator](../create/component-gen-overview.md) | [Micro Frontends](../../tutorials/create/mfe/) |
+| [Entando Blueprint Feature](../create/blueprint-features.md) | [Microservices](../../tutorials/create/ms/generate-microservices-and-micro-frontends.md) |
+| [Component Generation Technologies](../create/component-gen-tech.md) | [Build and Publish Project Bundles](../../tutorials/create/pb/publish-project-bundle.md) |
+
+## Curate and Share Components 
+On the Entando Platform, Curate and Share means development teams can bundle and share components within a central catalog. Multiple teams can share and collaborate on a single component or whole packaged business capabilities (PBCs) across organizations and applications. They can test and assess components for readiness, manage and publish versions, and share metadata for continuous upgrades.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/v7.0/docs/README.md:24]: Curate and Share -> just Curate -- we never use Share outside of the left nav, and Curate implies/includes that in the descriptions

"Multiple teams can share and collaborate on a single component..." -> "Multiple teams can collaborate on and share a single component..." --reverse order because chronologically you collaborate before sharing

remove "whole" -- unnecessary; a packaged business capability is by definition the whole kit and caboodle. you would never refer to a component as part of a business capability - instead you'd use the component name.

avoid "they"... users? teams? "they" is awkward when referring to a group here; "they" reads like people performing actions but refers to groups

File: vuepress/docs/v7.0/docs/README.md [hunk 5]
```diff
@@ -5,28 +5,56 @@ sidebarDepth: 0
...
+| [Entando Blueprint Feature](../create/blueprint-features.md) | [Microservices](../../tutorials/create/ms/generate-microservices-and-micro-frontends.md) |
+| [Component Generation Technologies](../create/component-gen-tech.md) | [Build and Publish Project Bundles](../../tutorials/create/pb/publish-project-bundle.md) |
+
+## Curate and Share Components 
+On the Entando Platform, Curate and Share means development teams can bundle and share components within a central catalog. Multiple teams can share and collaborate on a single component or whole packaged business capabilities (PBCs) across organizations and applications. They can test and assess components for readiness, manage and publish versions, and share metadata for continuous upgrades. 
+
+| Documentation | Tutorials        |
+| :------------ | :-------- |
+| [Bundle and Component Descriptors](../curate/ecr-bundle-details.md) | [Install Bundles from a Private Git Repository](../../tutorials/curate/ecr-private-git-repo.md) |
+| [Filtering Bundles](../curate/ecr-bundle-filters.md) | [Install Bundle Plugins from a Private Image Repo](../../tutorials/curate/ecr-private-images.md) |
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/v7.0/docs/README.md:29]: change to Filter? none of the other verbs use "ing"

File: vuepress/docs/v7.0/docs/README.md [hunk 6]
```diff
@@ -5,28 +5,56 @@ sidebarDepth: 0
...
+| [Customize Bundle Info in App Builder](../curate/ecr-bundle-presentation-config.md) |   |
+| [Bundle Version and Updates FAQ](../curate/ecr-bundle-versions-faq.md) |  |
+| [Bundle Uninstall](../curate/ecr-uninstall-flow.md) |  |
+
+
+## Compose an Application 
+The Compose stage is centered around the [Entando App Builder](../compose/app-builder.md), a low-code composition user interface. In the App Builder, application owners can assemble new components or upgrade existing ones with templates, widgets and PBCs. The [Entando Component Manager](../compose/ecm-overview.md) and [Entando Component Repository](../compose/ecr-overview.md) are integrated into the App Builder to manage your components, making the process flexible and agile.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/v7.0/docs/README.md:36]: "making the process flexible and agile" seems too wordy "for flexible and agile component management"? "flexible" and "agile" are almost overused on this page...

File: vuepress/docs/v7.0/docs/README.md [hunk 7]
```diff
@@ -5,28 +5,56 @@ sidebarDepth: 0
...
+| Documentation | Tutorials        |
+| :------------ | :-------- |
+| [Welcome Wizard](../compose/welcome-wizard.md) | [Page Management](../../tutorials/compose/page-management.md) |
+| [Entando Component Repository](../compose/ecr-overview.md) | [Widgets and Fragments](../../tutorials/compose/widgets-fragments.md) |
+| [Entando App Builder](../compose/app-builder.md) | [Content Creation](../../tutorials/compose/content-tutorial.md) |
+| [Entando Component Manager](../compose/ecm-overview.md) | [Content Templates](../../tutorials/compose/content-templates-tutorial.md)  |
+|  | [Digital Assets](../../tutorials/compose/digital-assets-tutorial.md) |
+
+## Consume Applications 
+In the development lifecycle of an application, Consume is the execution stage where users interact with the app. On Entando, the modular design of the components and services means they can be scaled individually, or as a whole, according to those interactions. Likewise, content can be updated, engine rules adapted, and upgrades made seamlessly. As a result, the development cycle is a continuous process with minimal disruptions and faster time to market.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/v7.0/docs/README.md:47]: would remover "the" in front of "components and services" -- unnecessary and an extra word in an already full sentence

"according to those interactions" is unnecessary; it's implied, and "those" interactions makes it seem like we're talking about specific interactions instead of any interaction

"upgrades made seamlessly" reads very awkwardly

"...a continuous process with...faster time to market" doesn't make sense... the process isn't going to market --"...minimal disruptions reducing time to market"

File: vuepress/docs/v7.0/docs/getting-started/README.md [hunk 8]
```diff
@@ -41,21 +29,20 @@ https://multipass.run/#install
 curl -sfL https://get.entando.org | bash
 ```
 
-3. The progress of the install will be displayed on the console. Installation can take 10+ minutes depending on how long the Docker images take to download. The sequence of steps performed by the CLI is identical to the manual steps below. To understand what the CLI does, review the manual steps.
+3. The progress of the install will be displayed on the console. Installation can take up to 10 minutes, depending on how long the Docker images take to download. The sequence of steps performed by the CLI is identical to the manual steps below. To understand what the CLI does, review the manual steps.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/v7.0/docs/getting-started/README.md:32]: will be displayed -> is displayed

10+ means more than ten

File: vuepress/docs/v7.0/docs/getting-started/README.md [hunk 9]
```diff
@@ -41,21 +29,20 @@ https://multipass.run/#install
...
 4. The URL to access the Entando App Builder will print to the console once the install completes.
 5. Login with username:`admin` and password: `adminadmin`. Refer to [Login to Entando](#login-to-entando) for more information and next steps.
 
 ## Manual Install
 
-Manual installation allows you to configure a shared cluster or customize a local developer environment. The following learn-as-you-go approach gives you a working knowledge of Kubernetes as you get Entando up and running in a local environment. This is a three stage process:
+Manual installation allows you to configure a shared cluster or customize a local developer environment. The following learn-as-you-go approach gives you a working knowledge of Kubernetes as you get Entando up and running in a local environment. After downloading the necessary files, your instance of Kubernetes will be up and running in <60 seconds.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/v7.0/docs/getting-started/README.md:38]: gives you a -> provides

too many "up and running"s here

<60 seconds is the guaranteed time frame based on empirical data?

File: vuepress/docs/v7.0/tutorials/README.md [hunk 10]
```diff
@@ -3,16 +3,17 @@ sidebarDepth: 2
...
-::: tip Entando simplifies the development of modern apps:
+::: tip The Entando Platform simplifies and automates the DevOps lifecycle of applications orchestrated by Kubernetes.
 
-1. Built using modern JavaScript frameworks
-2. Backed by microservices deployed in containers, and
-3. Orchestrated by Kubernetes for fully automated DevOps lifecycles
-:::
 
-Entando supports full stack micro frontend and microservice architectures for codebases that are easier to understand, maintain, and debug across large, distributed teams, and comes with a private component repository that makes it easy for teams to share and reuse components.
+* Build end-to-end modularity into your application for codebases that are easier to maintain & debug.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/v7.0/tutorials/README.md:9]: & -> and

File: vuepress/docs/v7.0/tutorials/README.md [hunk 11]
```diff
@@ -3,16 +3,17 @@ sidebarDepth: 2
...
+* Share and reuse components from a common catalog between teams and applications.
+* Leverage the JHipster blueprint, standardizing and accelerating development.
+
+:::
+ 
 
-Use the navigation on the left to find step-by-step tutorials for common tasks or check out our learning paths below for a more structured approach.
+Check out the learning paths below to start, with varying levels of experience and needs for the developer.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/v7.0/tutorials/README.md:16]: "to start" is awkward, like to start what?

File: vuepress/docs/v7.0/docs/README.md (hunk 6 above)
Comment (jyunmitch) [reply vuepress/docs/v7.0/docs/README.md:36]: Actually flexible is used just once and agile is used twice. This explains the original claim in the intro.

Comment (Lyd1aCla1r3) [reply vuepress/docs/v7.0/docs/README.md]: i always think of the service/tool Agile when i see it used as an adjective; hopefully this isn't the crowd to make that mistake

File: vuepress/docs/v7.0/docs/README.md (hunk 7 above)
Comment (Lyd1aCla1r3) [reply vuepress/docs/v7.0/docs/README.md]: saying you can scale individually or as a whole isn't sufficient? 

minimal disruptions would be a contributor to faster time to market... it's just that as-is the sentence reads "a continuous process with faster time to market" - because of the "and." i'd rephrase with a verb, like "and enables" to keep the two separate

File: vuepress/docs/v7.0/docs/README.md (hunk 5 above)
Comment (jyunmitch) [reply vuepress/docs/v7.0/docs/README.md]: No reason for it to match.

File: vuepress/docs/v7.0/docs/getting-started/README.md (hunk 1 above)
Comment (nshaw) [reply vuepress/docs/v7.0/docs/getting-started/README.md:57]: @jyunmitch looks good to me.

File: vuepress/docs/v7.0/docs/getting-started/concepts-overview.md [hunk 12]
```diff
@@ -1,173 +1,167 @@
 # Entando Architecture Concepts
 
-A portal, website, web app, or mobile app built with Entando is called an Entando application.
-An Entando application is an assembly of out of the box and/or custom built components running on the
-Entando Platform. Components can be widgets, micro frontends, microservices, page templates, WCMS content or WCMS content types.
+Entando simplifies the process for enterprises looking to modernize across on-prem and cloud infrastructures with native support for Javascript app development, micro frontend and microservices architecture, deployment of apps in scalable containers, and automated management of containers with Kubernetes. 
+
+This document explores the architecture of the platform and some notable runtime characteristics.
+
+![entando-architecture](../entando-architecture.png)
```
Comment (nshaw) [inline vuepress/docs/v7.0/docs/getting-started/concepts-overview.md:7]: This looks like a screenshot of the full slide from the presentation with left and right background images. I think we had that before but can we clean that up and just include the diagram itself? We can probably drop the header as well but then when you click on the image you just get the interesting parts.

File: vuepress/docs/v7.0/docs/getting-started/concepts-overview.md [hunk 13]
```diff
@@ -1,173 +1,167 @@
...
-these services are made possible by the communication with the Entando
-Kubernetes service, the only service of the Entando ecosystem (other
-than the operator itself) able to interact with the cluster and some of
-the Entando Custom Resources.
+The [Entando Component Manager](../compose/ecm-overview.md) lists the EntandoDeBundles accessible from the EntandoApp and provides the install/uninstall/upgrade/downgrade services to manage a bundle. The K8s service communicates with the cluster to provide these services. It is the only service, other than the operator, that can interact with the cluster and the required custom resources. 
 
 #### Entando Kubernetes Service
 
-The Entando Kubernetes Service is part of the Entando cluster
+The Entando K8s service is part of the cluster
```
Comment (nshaw) [inline vuepress/docs/v7.0/docs/getting-started/concepts-overview.md:98]: Minor, but maybe Kubernetes integration service per the glossary. I wouldn't shorten it here in the full name like this.

File: vuepress/docs/v7.0/docs/getting-started/concepts-overview.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (nshaw) [reply vuepress/docs/v7.0/docs/getting-started/concepts-overview.md:217]: Please open a new ticket for tackling this ingresses section. Some of it is out-of-date (no more /digital-exchange) or doesn't match OpenShift 4.8 (locations of resources). We may want to include this in the next interview with Engineering to see if we can strip it down.

File: vuepress/docs/v7.0/docs/getting-started/concepts-overview.md [hunk 14]
```diff
@@ -211,36 +205,36 @@ In this table you can see what are the default routes that are exposed for each
...
-This allows the micro frontend developers, who need access to the plugin, to disregard CORS policy and the full path where the plugin is exposed. The plugin can be referenced using relative URLs.
+This allows micro frontend developers, who need access to the plugin, to disregard CORS policy and the full path where the plugin is exposed. The plugin can be referenced using relative URLs.
 
-### How to Check Ingresses in my Cluster
+### How to Check Ingresses in My Cluster
 
 #### Using the Openshift Dashboard
 
-On the Openshift dashboard, ingresses are not exposed directly as pods and deployments. The dashboard provides direct access to the Ingress Routes (or simply routes) under the `Applications` → `Routes` menu.
+On the Openshift dashboard, ingresses are not exposed directly as pods and deployments. The dashboard provides direct access to the ingress routes under the `Applications` → `Routes` menu.
```
Comment (nshaw) [inline vuepress/docs/v7.0/docs/getting-started/concepts-overview.md:221]: Openshift -> OpenShift

File: vuepress/docs/v7.0/tutorials/README.md (hunk 10 above)
Comment (nshaw) [reply vuepress/docs/v7.0/tutorials/README.md:9]: I think this was missed. I still see the &.

**Review by nshaw — CHANGES_REQUESTED** [review]
Few requests and then apply to next as well.

File: vuepress/docs/next/tutorials/README.md [hunk 15]
```diff
@@ -7,18 +7,20 @@ Some screenshots or references to the previous version may be out-of-date and so
 only be available by building from source.
 :::
 
+
 # Learning Paths
 
-::: tip Entando simplifies the development of modern apps:
+::: tip Entando simplifies and automates the DevOps lifecycle of applications orchestrated by Kubernetes.
```
Comment (nshaw) [inline vuepress/docs/next/tutorials/README.md:13]: This is a bit too narrow since the 4Cs cover more than just DevOps. Maybe "Entando simplifies the development of composable applications deployed on Kubernetes."  I actually don't love these bullets but let's keep those for now.

File: vuepress/docs/next/tutorials/README.md [hunk 16]
```diff
@@ -7,18 +7,20 @@ Some screenshots or references to the previous version may be out-of-date and so
...
-Entando supports full stack micro frontend and microservice architectures for codebases that are easier to understand, maintain, and debug across large, distributed teams, and comes with a private component repository that makes it easy for teams to share and reuse components.
+* Build end-to-end modularity into your application for codebases that are easier to maintain and debug.
+* Share and reuse components from a common catalog between teams and applications.
+* Leverage the JHipster blueprint, standardizing and accelerating development.
+
+:::
+ 
 
-Use the navigation on the left to find step-by-step tutorials for common tasks or check out our learning paths below for a more structured approach.
+Check out the learning paths below, for varying levels of experience and needs for the developer.
```
Comment (nshaw) [inline vuepress/docs/next/tutorials/README.md:23]: I think we discussed improving this - "varying levels of experience and needs" is awkward to me. Maybe "Check out the learning paths below to find step-by-step tutorials for the most common tasks you may encounter."

**Review by nshaw — CHANGES_REQUESTED** [review]

File: vuepress/docs/next/tutorials/README.md (hunk 15 above)
Comment (jyunmitch) [reply vuepress/docs/next/tutorials/README.md:13]: I took out the blue highlighted box because it always bothered me and we don't use it anywhere else. What about the bullets needs improvement?  It feels wrong to publish when something is off on a page that will get the more traffic than most pages.

**Review by nshaw — APPROVED** [review]

File: vuepress/docs/next/docs/README.md [hunk 17]
```diff
@@ -12,28 +12,56 @@ only be available by building from source.
...
 
-With native support for Javascript app development, a micro frontend and microservices architecture, deployment of apps in containers that can be individually scaled up and down, and automated management of containers with Kubernetes, we simplify the move for enterprises looking to modernize across on-prem and cloud infrastructures.
+It is a distributed and composable environment in which components can be built, scaled and iterated independently, helping teams build agile applications that can adapt to change.
 
-## Entando Architecture
+![Entando Platform Development Process](./getting-started/img/concepts.png) 
 
-![entando-architecture](./entando-architecture.png)
+On the Entando Platform, development of an application is organized into four stages: Create, Curate, Compose, and Consume. In the simplest terms, you create functional building blocks, curate them into a catalog, compose the blocks onto an app or website, which is then consumed by a user.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/docs/README.md:19]: the second sentence doesn't make sense as-is. simplest option is to add "and" before "compose the blocks." without "and" it's grammatically incorrect.

File: vuepress/docs/next/docs/README.md [hunk 18]
```diff
@@ -12,28 +12,56 @@ only be available by building from source.
...
 
-The Entando platform includes the following major elements:
+## Create Components 
+The first step in building a modern application on Entando is to design and create each independent component or functional block. Developers can use the [Entando Component Generator](./create/component-gen-overview.md) and [Blueprint](./create/blueprint-features.md) to automate the building and packaging of project files, with the flexibility to customize while using software they are accustomed to. The modular design of apps built on Entando means new business capabilities can be integrated into your application without reworking existing systems.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/docs/README.md:22]: should be consistent with either "they" or "you," not mix these

File: vuepress/docs/next/docs/README.md [hunk 19]
```diff
@@ -12,28 +12,56 @@ only be available by building from source.
...
-- (optional) `Entando Business Automation Bundle`: workflow and task automation including out of the box integration with Red Hat Process Automation Manager (PAM).
+| Documentation | Tutorials        |
+| :------------ | :--------- |
+| [Entando Component Generator](./create/component-gen-overview.md) | [Micro Frontends](../tutorials/create/mfe/) |
+| [Entando Blueprint Feature](./create/blueprint-features.md) | [Microservices](../tutorials/create/ms/generate-microservices-and-micro-frontends.md) |
+| [Component Generation Technologies](./create/component-gen-tech.md) | [Build and Publish Project Bundles](../tutorials/create/pb/publish-project-bundle.md) |
 
-## Why Kubernetes?
+## Curate and Share Components 
+On the Entando Platform, Curate and Share means development teams can bundle and share components within a central catalog. Multiple teams can collaborate on a single component or packaged business capabilities (PBCs) and share them across organizations and applications. They can test and assess components for readiness, manage and publish versions, and share metadata for continuous upgrades.
```
[the remark this answers is not in the document]
Comment (jyunmitch) [reply vuepress/docs/next/docs/README.md:31]: Share is part of the menu title and need to be consistent with that. Also, the items under that menu header don't really make sense without the Share.

Single is there for emphasis. Can we not belabor this point and sometimes disagree.

**Review by nshaw — APPROVED** [review]
